tests/fsharpqa/Source/CodeGen/EmittedIL/TestFunctions/Testfunction22h.fs

```diff
@@ -1,6 +1,37 @@
 // #NoMono #NoMT #CodeGen #EmittedIL   
 #light
-try 
-   System.Console.WriteLine() 
-with
-   | _ -> System.Console.WriteLine()
+let test1() =
```

Comment:
Lol ... I'm trying to kill fsharpqa and you are adding more tests to it :-).
It's fine, when I get to the emitted il tests I will move them.
Kevin